guide/how-it-works/ecash/cashu.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: guide
+title: Cashu
+description: Learn the basics of Cashu.
+nav_order: 2
+parent: Ecash
+grand_parent: How it works
+permalink: /guide/how-it-works/ecash/cashu/
+redirect_from:
+ - /guide/ecash/cashu
+image: https://bitcoin.design/assets/images/guide/how-it-works/ecash/ecash.jpg
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+Explains generally how multi-key schemes work.
+
+Illustration sources
+
+https://www.figma.com/community/file/888680264445459448
+https://www.figma.com/community/file/995256542920917246/BDG---Private-key-management-illustrations
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/ecash.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/ecash@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/ecash-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/ecash-mobile@2x.jpg"
+   alt-text = "ecash overview illustration"
+   width = 1600
+   height = 700
+   layout = "full-width"
+%}
+
+# Cashu
+Cashu is an ecash protocol on that interoperates with the Lightning Network. Unlike Fedimint, which uses a federated model to distribute trust among multiple entities, Cashu operates with independent mints, offering greater flexibility and control to users. This makes Cashu distinct in providing an easy to deploy, flexible, and agile approach to managing Ecash on bitcoin.
```

Comment:
> federated model to distribute trust among multiple entities, Cashu operates with independent mints

Don't think it is quite right. Both systems have independent mints. It's just that Fedimint has multiple guardians per mint, and Cashu has one guardian per mint (or maybe I a am wrong here...).

> offering greater flexibility and control to users

Would like to hear more about this.

Reply:
Yeah, this is an oversight on my part. I think this Fedi article does a good job of explaining the different types of Fedimints: https://www.fedi.xyz/blog/understanding-the-different-types-of-fedimint-setups
I'll update so it's clearer that Fedimint allows for a Federated (multiple guardians) and an Unfederated mint (Solo mint).

I'll removed the 'offering greater flexibility and control to users' and stick with saying that it's easier to deploy. The more I think about claiming that independent mints offer greater flexibility the more I think it's wrong. What I'm trying to say is that it's easier to deploy and make changes to.

Reply:
Is the easier to deploy aspect just a matter of implementation? Like Cashu having spend more time creating a simple installer?

Reply:
It's easier to deploy in my experience. I tried to deploy a Federated mint and I was unable to get the lightning gateway (another dependency needed to setup a lightning compatibility with the federation). I even took a 1 hour call with a fedimint dev and we were still unable to set it up. With Cashu the installation is much easier, you just run some commands and install a python app, configure a text file, and your Cashu mint is up and running and it supports lightning out of the box.

Reply:
I was using the term "independent" mint to refer to a Cashu mints. It's true that Fedi doesn't use that term anywhere in their docs. In their offical document I only find the word "Federated Mint" but in the Fedi link they use the words "Unfederated Mints (Solo Mints)" and "Federated Single Guardian Mints" but those are more descriptors of a federations types I guess.